### PR TITLE
Update create-user-accounts-with-microsoft-365-powershell.md

### PR DESCRIPTION
**Pull request by msbemba**
Added Licensing SKU document in a couple of places to avoid delay in finding the SKU

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/8781

File: microsoft-365/enterprise/create-user-accounts-with-microsoft-365-powershell.md [hunk 1]
```diff
@@ -35,7 +35,7 @@ When you create user accounts in PowerShell, certain account properties are alwa
...
 |**FirstName** <br/> |No  <br/> ||
 |**LastName** <br/> |No  <br/> ||
-|**LicenseAssignment** <br/> |No  <br/> |This is the licensing plan (also known as the license plan or SKU) from which an available license is assigned to the user account. The license defines the Microsoft 365 services that are available to the account. You don't have to assign a license to a user when you create the account, but the account must have a license to access Microsoft 365 services. You have 30 days to license the user account after you create it. |
+|**LicenseAssignment** <br/> |No  <br/> |This is the licensing plan also known as the [license plan or SKU](https://docs.microsoft.com/azure/active-directory/enterprise-users/licensing-service-plan-reference) from which an available license is assigned to the user account. The license defines the Microsoft 365 services that are available to the account. You don't have to assign a license to a user when you create the account, but the account must have a license to access Microsoft 365 services. You have 30 days to license the user account after you create it. |
```
Comment (JohanFreelancer9) [inline microsoft-365/enterprise/create-user-accounts-with-microsoft-365-powershell.md]:
```suggestion
|**LicenseAssignment** <br/> |No  <br/> |This is the licensing plan (also known as the [license plan or SKU](/azure/active-directory/enterprise-users/licensing-service-plan-reference)) from which an available license is assigned to the user account. The license defines the Microsoft 365 services that are available to the account. You don't have to assign a license to a user when you create the account, but the account must have a license to access Microsoft 365 services. You have 30 days to license the user account after you create it. |
```

File: microsoft-365/enterprise/create-user-accounts-with-microsoft-365-powershell.md [hunk 2]
```diff
@@ -81,7 +81,7 @@ New-MsolUser -DisplayName <display name> -FirstName <first name> -LastName <last
 >PowerShell Core doesn't support the Microsoft Azure Active Directory Module for Windows PowerShell module and cmdlets that have *Msol* in their name. Run these cmdlets from Windows PowerShell.
 >
 
-To list the available licensing plan names, use this command:
+To list the available [licensing plan names](https://docs.microsoft.com/azure/active-directory/enterprise-users/licensing-service-plan-reference), use this command:
```
Comment (JohanFreelancer9) [inline microsoft-365/enterprise/create-user-accounts-with-microsoft-365-powershell.md]:
```suggestion
To list the available [licensing plan names](/azure/active-directory/enterprise-users/licensing-service-plan-reference), use this command:
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@kelleyvice-msft Please merge this PR. Thanks

**Pull request merged**